src/lib/services/repository-access/RepositorySearchService.ts

```diff
@@ -37,23 +38,31 @@ export class MultipleRepositorySearchService {
         });
         const log = Log.create();
         const prismaConnector = PrismaConnector.create();
-        return new MultipleRepositorySearchService(repositoryClient, submodelRepositoryClient, prismaConnector, log);
+        return new RepositorySearchService(repositoryClient, submodelRepositoryClient, prismaConnector, log);
     }
 
     static createNull({
         shellsSavedInTheRepositories = [],
         submodelsSavedInTheRepository = [],
         log = null,
-    }: NullableMultipleDataSourceSetupParameters = {}): MultipleRepositorySearchService {
+    }: NullableMultipleDataSourceSetupParameters = {}): RepositorySearchService {
         const aasUrls = [...new Set(shellsSavedInTheRepositories?.map((entry) => entry.repositoryUrl))];
-        return new MultipleRepositorySearchService(
+        return new RepositorySearchService(
             AssetAdministrationShellRepositoryApi.createNull({ shellsSavedInTheRepositories }),
             SubmodelRepositoryApi.createNull({ submodelsSavedInTheRepository }),
             PrismaConnector.createNull({ aasUrls }),
             log ?? Log.createNull(),
         );
     }
 
+    async getAasFromDefaultRepository(aasId: string): Promise<AssetAdministrationShell> {
+        try {
+            return await this.repositoryClient.getAssetAdministrationShellById(aasId);
+        } catch (e) {
+            throw new Error('AAS not found');
```

Comment:
We do have a ```NotFoundError```, would be better to use this one as opposed to using the generic Exception